alfresco-remote-api-exceptions/build.gradle

```diff
@@ -5,6 +5,10 @@ plugins {
 group = rootProject.group
 version = rootProject.version
 
+repositories {
```

Comment:
not necessary?

Reply:
I get warnings trying to resolve lombok if I don't have this.